examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
I've always found this just slows you down. You would like the examples to be built and tested, so they should be built with the main project. And you'll have to update the examples so they compile with new versions, so they might as well follow the version they're an example of…? If they are built with the main project, you get everything updated for you with versions:set and release:prepare. Keeping them on a separate version is more cumbersome.

(But they should be tied to the apollo-http-service version they're using when we'll start versioning the api separate from modules and assemblies.)